code/modules/projectiles/projectile.dm

```diff
@@ -502,7 +504,7 @@
 		var/matrix/M = new
 		M.Turn(Angle)
 		transform = M
-	if(homing)
+	if(homing && world.time >= next_homing_process) //Nsv13 performance enhancements

```

Comment:
you might need a (world.time >= next_homing_process)) not sure though

Reply:
Those evaluate the exact same